core/db/default/spree/countries.rb

```diff
@@ -16,4 +16,4 @@
   Spree::Country.create!(countries)
 end
 
-Spree::Config[:default_country_id] = Spree::Country.find_by(name: "United States").id
+Spree::Config[:default_country_id] = Spree::Country.find_by(iso: "US").id
```

Comment:
Shouldn't we change this to `||=` because without that the US is always the default country while seeding?